README.md

```diff
@@ -25,7 +25,44 @@ uv add cicerone
 
 ## Quick Start
 
-```sh
-# Generate an object from an OpenAPI schema
-TODO
+```python
```

Comment:
Rename each `spec` to be something like `file_spec` and `url_spec` etc. Separate this into a code block demonstrating the different parsing options.